package.json

```diff
@@ -96,7 +96,7 @@
 		"install-if-no-packages": "node bin/install-if-no-packages.js",
 		"lint": "run-s -s 'lint:*'",
 		"lint:config-defaults": "node bin/validate-config-keys.js",
-		"lint:css": "stylelint \"**/*.{css,scss}\"",
+		"lint:css": "stylelint --rd --risd \"**/*.{css,scss}\"",
```

Comment:
Can we enable these in the config file instead, so that the IDE can report these too? I think this might do it: https://stylelint.io/user-guide/usage/options/#reportdescriptionlessdisables and https://stylelint.io/user-guide/usage/options/#reportdescriptionlessdisables

But this is an excellent find, thanks for adding it!

Reply:
Good suggestion. I hadn't thought about that. Thanks